pandas/core/reshape.py

```diff
@@ -1126,6 +1130,12 @@ def get_dummies(data, prefix=None, prefix_sep='_', dummy_na=False,
     1  0  1    0
     2  0  0    1
 
+    >>> pd.get_dummies(s1, dummy_na='keep')
```

Comment:
These are already there as I stated above. See the screen shot here

<img width="289" alt="screen shot 2017-04-07 at 9 32 06 am" src="https://cloud.githubusercontent.com/assets/5296416/24804730/17094aee-1b75-11e7-9162-b028138178b1.png">